openupgrade_scripts/scripts/web/15.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,12 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_xmlids(
+        env.cr,
+        [
+            ("web.external_layout_background", "web.external_layout_striped"),
+            ("web.external_layout_clean", "web.external_layout_bold"),
+        ],
```

Comment:
```suggestion
            ("web.report_layout_background", "web.report_layout_striped"),
            ("web.report_layout_clean", "web.report_layout_bold"),
        ],
```
also report layouts?

Reply:
My omission, I have added